src/popup.css

```diff
@@ -35,6 +35,6 @@ a.button {
     margin-bottom: 5px;
 }
 
-.item {
+.deal {
     display:inline-block;
 }
```

Comment:
Do you like the idea of shrinking the grey border on the carousel controls? I think it makes the popup look cleaner, since it exposes more white background. Here's a screenshot with the smaller controls (and the font changes mentioned above):

![image](https://user-images.githubusercontent.com/11142171/38850066-8c5a509c-41c4-11e8-9ce1-4896ecf38c35.png)

I was able to do this by adding the following code:

```
.carousel-control.left, .carousel-control.right {
    max-height: 100px;
    display: inline-block;
    position: absolute;
    top: 50%;
    z-index: 5;
    margin-top: -50px;
}
```

Is there a cleaner way of doing this? That css looks messy